libr/core/rvc.c

```diff
@@ -227,9 +228,26 @@ static RList *get_commits(const char *rp, const size_t max_num) {
 	return ret;
 }
 
-static bool update_blobs(RList *blobs, const RList *nh) {
+
+static bool in_rvc_ignore(const char *rp, const char *rpf) {
+	char *ignorep = r_str_newf ("%s" R_SYS_DIR IGNORE_NAME, rp);
+	if (ignorep) {
+		char *content = r_file_slurp (ignorep, 0);
+		free (ignorep);
+		if (content) {
+			bool ret = strstr (content, rpf);
```

Comment:
Free content

Reply:
Better cache the contents in rvector or rlist . Use rstr.split and just check for the given string not for strstr which may result on false positives

Reply:
I don't understand why you would want to cache it (in memory) in_rvc_ignore is only ran once per ravc2 command